src/Indodana.php

```diff
@@ -26,13 +21,13 @@ class Indodana
   public function __construct(array $config = [])
   {
     $config = array_merge([
-      'isProduction' => false
+      'livemode' => false
```

Comment:
We use `sdkEnvironmentStage` in our [ios SDK](https://github.com/indodana/indodana-ios-whitelabel-sdk/blob/master/Sources/SDKEnvironmentStage.swift) better use that because u can add, 
* dev + staging (for development) 
* sandbox 
* prod

Reply:
How about `environmentStage`? IMO `sdkEnvironmentStage` is too verbose

Reply:
Btw, actually we should only allow the user of this SDK to `sandbox` (we say it testing mode) and `prod` (we say it live mode). We shouldn't allow another environment right?

Reply:
actually you're right, how about make it explicit with enum?
* sandbox
* production

Reply:
After thinking again about this. I'm more into `environment`:

Reply:
After discussing with Sendy (with the concern on communication), we decide to stick with `SANDBOX` and `PRODUCTION` as the environment.
In addition, I add `getAvailableEnvironments()` which will fetch environments available on Indodana. Cc: @sendyhalim